torch/_inductor/codegen/triton.py

```diff
@@ -1082,10 +1080,12 @@ def print_min(expr):
             # It'd be less # error-prone to use and/or/not, which is suported by triton
             cond = f"((0 <= {var}) & ({var} < {size}))"
             cond_print = f"0 <= {var} < {size}"
-            if not isinstance(original_index, sympy.Integer):
```

Comment:
This condition was unnecessary, as it's never going to be true when we have indirect indexing.